debian/snapd.postinst

```diff
@@ -11,4 +11,13 @@ case "$1" in
         if dpkg --compare-versions "$2" lt-nl "2.0.7"; then
             ldconfig
         fi
+        # start required services
+        if [ -d /run/systemd/system ]; then
+            systemctl start snapd.boot-ok.service
+            systemctl start snapd.refresh.timer
+            systemctl start snapd.refresh.service
+            systemctl start snapd.autoimport.service
+            systemctl start snapd.socket
+            systemctl start snapd.service
```

Comment:
Also, given you're doing it manually, could you please do a single call to systemctl instead of N? It'll happily take all the unit files you want to perform the same action in a single call